geoips/plugins/modules/readers/sfc_winds_text.py

```diff
@@ -12,6 +12,7 @@
 interface = "readers"
 family = "standard"
 name = "sfc_winds_text"
+source_names = ["sar-spd", "smap-spd", "smos-spd", "amsr2"]
```

Comment:
This is another one that could be somewhat problematic.  Any geoips-supported wind data could be written out, and any of those sources could potentially be read back in.

Reply:
Though - there is a hard coded mapping in the reader, so these are explicitly the only sources that are supported.  So don't worry about this one.

Reply:
I won't worry about this one for the time being. If anything changes in the future, let me know and I can make the appropriate adjustments. Luckily this information is only used by the CLI and shouldn't cause any problems internally to GeoIPS.